src/com/esotericsoftware/kryo/serializers/CompatibleFieldSerializer.java

```diff
@@ -135,9 +135,14 @@ public T read (Kryo kryo, Input input, Class<? extends T> type) {
 				} catch (KryoException ex) {
 					String message = "Unable to read unknown data (unknown type). (" + getType().getName() + "#" + cachedField + ")";
 					if (!chunked) throw new KryoException(message, ex);
-					if (DEBUG) debug("kryo", message, ex);
-					inputChunked.nextChunk();
-					continue;
+                    try {

```

Comment:
Please correct the indentation of this block.

Reply:
Done here. It's no difference in my IDEA editor but show wrong format in the page. LOL